tx-pool/Cargo.toml

```diff
@@ -32,6 +32,7 @@ ckb-network = { path = "../network", version = "= 0.100.0-pre" }
 ckb-channel = { path = "../util/channel", version = "= 0.100.0-pre" }
 ckb-traits = { path = "../traits", version = "= 0.100.0-pre" }
 sentry = { package = "ckb-sentry", version = "0.21.0", optional = true }
+dotenv = "0.15.0"
```

Comment:
It's better to add `dotenv` only in `ckb-bin` and inject options only in that crate.

Reply:
Please ignore .env in the git, and add a .env.example

Reply:
We will not be able to use this in other crates' unit tests If we only load the `dotenv` file in `ckb-bin`.

static START: Once = ONCE_INIT;

pub fn var<K: AsRef<OsStr>>(key: K) -> Result<String> {
    START.call_once(|| {
        dotenv().ok();
    });
    std::env::var(key)
}